### PR TITLE
Fixed broken links on the page with relative links.

### PR DESCRIPTION
**Pull request by swarna04**
- Fixed broken links on the page.
- Used relative links (helps with migration).
- Explicitly specified references to extension README.md.

### CODE REVIEW

File: resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 {% endtab %}
 {% endtabs %}
 
-In-depth instructions can be seen at the [Analytics Readme](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/b7706ec53f082a5385a6eb5870418af7c116f0b1/using-mobile-extensions/adobe-analytics/readme.md#add-analytics-to-your-app).
+In-depth instructions can be seen at the [Analytics Readme](../../using-mobile-extensions/adobe-analytics/README.md#add-analytics-to-your-application).
```
Comment (emdobrin) [inline resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md:134]: Let's reword this to:
`For more details, follow the steps for [adding Analytics to your application](../../using-mobile-extensions/adobe-analytics/README.md#add-analytics-to-your-application).`

**Review by emdobrin — APPROVED** [review]
Looks good with some rewording.

File: resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md (hunk 1 above)
Comment (swarna04) [reply resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md]: Updated as 
```
For more details, see [Add Analytics to your application](...).
```
